packages/types/src/base64.ts

```diff
@@ -19,10 +18,11 @@ export const InnerBase64Schema = z.object({ inner: Base64StringSchema });
 
 export const base64ToUint8Array = (base64: string): Uint8Array => {
   const validated = validateSchema(Base64StringSchema, base64);
-  const buffer = Buffer.from(validated, 'base64');
-  return new Uint8Array(buffer.buffer, buffer.byteOffset, buffer.byteLength);
+  const binString = atob(validated);
+  return Uint8Array.from(binString, m => m.codePointAt(0)!);
```

Comment:
Can we confirm this will never throw? -> `m.codePointAt(0)!`

If not, think it's better to handle:

```typescript

  return Uint8Array.from(binString, m => {
    const codePoint = m.codePointAt(0);
    if (!codePoint) throw new Error('Invalid base64 string');
    return codePoint;
  });
```

Reply:
`atob` will only throws [InvalidCharacterError](https://developer.mozilla.org/en-US/docs/Web/API/atob#invalidcharactererror) for an invalid base64 encoding, which will be caught by `validateSchema`. 

We're asserting that `m.codePointAt(0)!` is non-null, but if `binString` is an empty string, then it'll be undefined and throw a runtime error. Can you have base64 encoded empty strings?

I think we should handle for errors regardless.

Reply:
Hm... I don't think it's possible that this could throw. As @TalDerei pointed out, the string is already validated as Base64. Then, `Uint8Array.from()` is iterating over each character and calling `.codePointAt(0)` on it. Since it's a 1-character string, `.codePointAt(0)` would only return `undefined` if the character was somehow a byte that doesn't map to a Unicode code point. But since it's Base64, we know it does.